flake.lock

```diff
@@ -228,6 +228,22 @@
         "treefmt-nix": "treefmt-nix"
       }
     },
+    "runtimeDeps": {
```

Comment:
This input will be dropped after the upstream flake is updated in https://github.com/nix-community/nixpkgs-update/pull/419.